command/var_list_test.go

```diff
@@ -95,11 +97,11 @@ func TestVarListCommand_Offline(t *testing.T) {
 	}
 }
 
-// TestVarListCommand_Online contains all of the tests that use a testagent.
-// They reuse the same testagent so that they can run in parallel and minimize
+// TestVarListCommand_Online contains all of the tests that use a testServer.
+// They reuse the same testServer so that they can run in parallel and minimize
 // test startup time costs.
 func TestVarListCommand_Online(t *testing.T) {
-	ci.Parallel(t)
+	//ci.Parallel(t)
```

Comment:
Should this line just be deleted, or was it commented-out during test debugging?

Reply:
I was having local challenges running all of the tests without commenting out parallelism on this one test. The odd thing is, it's set up with its own `testServer` so I am now scratching my head about that...